client/src/views/Game.vue

```diff
@@ -73,6 +74,18 @@
               <p>
                 <v-switch v-model="pane_lock" :label="`Unlock Players Pane`"></v-switch>
               </p>
+
+              <!-- Need Help? button -->
+              <v-btn @click.native="helpMenu = !helpMenu" >Need Help?</v-btn>
+              <v-card v-show="helpMenu" class="mt-5 pa-2" outlined tile >                  
+                <router-link to="/help#rules">Rules</router-link> |
+                <router-link to="/help#tutorials">Tutorials</router-link> |
+                <router-link to="/help#cardAbilities">Card Abilities</router-link> |
```

Comment:
So, it looks like Vue supports the ability to open the link on a new page. which would actually be stellar.

`<router-link :to="{ name: 'fooRoute'}" target="_blank">` You would just need to add in the `target="_blank"`. which would allow for a new page to pop open.

This would prevent the player from navigating away from the game.

Reply:
Sorry to keep throwing random things on you to do here. if you need to engage in something else (like work). I will understand.

Reply:
Lame.. never mind. by opening in a new tab. it looses the anchor. I'd say leave it as is. and if we need to upgrade it later we can.

Reply:
Oh really! I thought about the new tab thing but that's interesting that it losses the anchor. haha